### DISCUSSION TITLE
anybody up for implementing stylesheets?

### DISCUSSION THREAD

**Discussion by eldeesmith** [mmmrqs/bl_ui_widgets#8] [Ideas]
This framework is extremely useful but the setup is super tedious, especially for a large project- which I think is a deterrent for wider adoption. 

Being able to create a single stylesheet and apply it to multiple UIs with minimal effort would be a godsend. I'm thinking a dynamic class that can parse html/css and then handle the appropriate BL_UI_* boilerplate? This way it could co-exist with the existing IMGUI style syntax but also allow for bulk UI creation, and even community sharing or templates.

**Comment by mmmrqs** [comment]
That would be very welcome.  It is beyond my skills but @SuddenDevelopment mentioned he was going to come up with such feature. Please check this other discussion: https://github.com/mmmrqs/bl_ui_widgets/issues/7#issue-1532802183
